packages/site/cms-site/src/sitePreview/PreviewParams.ts

```diff
@@ -0,0 +1,25 @@
+import { NextApiRequest } from "next";
+
+export type BlockPreviewParams = {
+    host: string;
+    path: string;
+};
+export type PreviewParams = BlockPreviewParams & {
+    previewSettings: {
+        includeInvisible: boolean;
+    };
+};
+
+export function getPreviewParamsFromReq(req: NextApiRequest): PreviewParams {
```

Comment:
We should call this `getSitePreviewParamsFromReq` to differentiate between site preview and block preview. Also, we `PreviewParams` should be `SitePreviewParams`